src/main/java/uk/gov/hmcts/sptribs/caseworker/event/ReinstateCase.java

```diff
@@ -80,12 +80,15 @@ public SubmittedCallbackResponse reinstated(CaseDetails<CaseData, State> details
         messageLine2.append("%n##  A notification will be sent via email to: ");
         if (!CollectionUtils.isEmpty(cicCase.getNotifyPartySubject())) {
             messageLine2.append("Subject, ");
+            cicCase.setNotifyPartySubject(null);
         }
```

Comment:
Just curious - if it's not empty, you append something, then you set it to null - why?

Reply:
for using same objects in other notify recipient actions, like send order